src/firmware/RC_firmware/RC_firmware.ino

```diff
@@ -0,0 +1,233 @@
+/* Firmware for Elsa
+   Test version
+   Author: Vincent Yuan, Modified: Nick Wu
+     Modified: James Asefa
+   Date: Oct 30, 2016
+*/
```

Comment:
Just nitpicky stuff, it would be a bit cleaner if written like:
```
/* Firmware for Elsa
   Author: Vincent Yuan, 
   Modified: Nick Wu
   Modified: James Asefa
   Date Last Modified: Oct 30, 2016
*/